addons/stock/views/stock_quant_views.xml

```diff
@@ -37,28 +37,43 @@
         <field name="model">stock.quant</field>
         <field eval="10" name="priority"/>
         <field name="arch" type="xml">
-            <form string="Inventory Valuation" create="false" edit="false" delete="false">
+            <form string="Inventory Valuation" create="false" delete="false">
                 <sheet>
                     <div class="oe_button_box" name="button_box">
                         <button class="oe_stat_button" icon="fa-exchange" type="object" name="action_view_stock_moves" string="Product Moves"/>
                     </div>
                     <group>
                         <group>
-                            <field name="product_id"/>
-                            <field name="location_id"/>
-                            <field name="lot_id" groups="stock.group_production_lot"/>
-                            <field name="package_id" groups="stock.group_tracking_lot"/>
-                            <field name="owner_id" groups="stock.group_tracking_owner"/>
+                            <field name="id" invisible="1"/>
```

Comment:
i validated with lap: keep the form as before! they're not reachable in inventory mode without custo so let's try to keep the diff minimal